source/ways-of-working/environments/external-access.html.md.erb

```diff
@@ -0,0 +1,102 @@
+---
+title: External Access in Demo
+last_reviewed_on: 2022-03-30
+review_in: 6 months
+weight: 10
+---
+
+# <%= current_page.data.title %>
+
+The demo environment is used to demonstrate new features to external stakeholders. 
+
+For guidance on how to deploy an application to the demo environment, see [Common Pipeline](https://hmcts.github.io/ways-of-working/common-pipeline/common-pipeline.html#building-infrastructure-in-the-demo-environment).
+
+There are three different ways in which an application in demo can be made available to external users. This is dependent on the ingress class used in your application.
+
+[Traefik](https://traefik.io/traefik/) is used as the ingress controller in HMCTS applications.
+
+The following traefik instances are available to use:
+
+| IngressClass | Default | Application configuration |
+|-|-|-|
+| traefik | Yes | Application will be reachable externally without VPN. End user will be required to authenticate to Azure AD before being redirected to the application |
+| traefik-private | No | Application will be available over VPN only |
+| traefik-no-proxy | No | Application will be reachable externally without VPN or authentication |
+
+### Protecting applications with Azure AD authentication
+    
+To ensure end users must authenticate to Azure AD before they can view the application in their web browser, you must also set the `enableOAuth` flag to `true`
+
+```yaml
+nodejs:
+    ingressClass: traefik
+    disableTraefikTls: false
+    enableOAuth: true
```

Comment:
this doesn't render very well with 4 spaces, use 2 instead
```suggestion
  ingressClass: traefik
  disableTraefikTls: false
  enableOAuth: true
```

Reply:
Done